### PR TITLE
Sandpack

### PR DESCRIPTION
**Pull request by jeetiss**
test sandpack

### CODE REVIEW

File: pages/sandpack.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+export default function App() {
+  return (
+    <>
+      <Head>
+        <meta
+          httpEquiv="Content-Security-Policy"
+          content="frame-src https://*;"
+        />
+        <style
```
Comment (danilowoz) [inline pages/sandpack.js]: This must be on [`pages/_document.tsx`](https://sandpack.codesandbox.io/docs/guides/ssr#nextjs)

Comment (jeetiss) [reply pages/sandpack.js]: Why doesn't it work this way? i don't wanna include sandpack styles for every page

Comment (danilowoz) [reply pages/sandpack.js]: As far as I know, Next.js make you use this way if you want to inject global CSS. Maybe you can conditionally render it based on the router?

File: app/sandpack/global-styles.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import { getSandpackCssText } from "@codesandbox/sandpack-react";
+import { useServerInsertedHTML } from "next/navigation";
+
+export const GlobalSandpackStyles = ({ children }) => {
+  useServerInsertedHTML(() => {
```
Comment (danilowoz) [inline app/sandpack/global-styles.js:7]: 😮 AMAZING!

Comment (jeetiss) [reply app/sandpack/global-styles.js:7]: yes, this hook makes the job in `app` folder

**Pull request merged**